dist/index.html

```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>To Do List</title>
-  <script defer src="main8bb1afef5bef7313d677.js"></script></head>
+  <script defer src="main5bd35c8d514c0628ef6e.js"></script></head>
```

Comment:
As specified in the requirements, kindly ensure that changes are saved ti the local storage when a task is marked as completed.

![image](https://github.com/melybee-07/To-Do-List/assets/20554583/7096701b-a750-485b-9aeb-a5c4c1e88678)